src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
So the commit from #2125 that brought this up wasn't a critical thing, it was just @markelog finding a guard he thought wasn't needed. We discussed it quite a bit there and *thought* there weren't any cases where it was needed, but we were wrong. So we either need to back out the original change or make further changes that we think would *really* prevent it from being needed.